javascript/javascript_in_the_real_world/form_validation_with_javascript.md

```diff
@@ -42,7 +42,7 @@ The form doesn't need to actually submit, but you should give an error message i
 3. Write the simple form elements.
 4. Add the JavaScript code that checks validation as the user progresses through the form.  When a user leaves a form field, it should automatically validate that field.
 5. Test out all possible cases.
-6. Don't forget to style validations with CSS. You can go through our [Form Validation lesson](https://www.theodinproject.com/lessons/intermediate-html-and-css-form-validation) for a refresher.
+6. Don't forget to style validations with CSS. You can go through our [Form Validation lesson](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-form-validation) for a refresher.
```

Comment:
Users on the Ruby path will suffer the same issue with this link being updated to the Node path. Rather than link to this lesson, where really only a small section is relevant to this instruction, what if instead we just said:

```suggestion
6. Don't forget to style validations with CSS by using the `:valid` and `:invalid` psuedo-classes!
```

Reply:
You're right. I didn't realize that both paths used the same code for this page. Your suggestion is better, thanks!